.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "testing-overhead/spring-petclinic-rest"]
+	path = testing-overhead/spring-petclinic-rest
+	url = https://github.com/spring-petclinic/spring-petclinic-rest.git
```

Comment:
How about using https://github.com/spring-petclinic/spring-petclinic-microservices ? That project already has a jmeter test script that we can reuse. And it seems to be more actively developed.

Reply:
For start spring-petclinic-rest which starts only once service is probably easier to manage than spring-petclinic-microservices which starts multiple services.

Reply:
I think it's too many components for our usage, and the service itself is fragmented across multiple services. I like petclinic-rest because you can do all the common workflows (create a pet, schedule an appointment, etc) with one app.